pycbc/waveform/compress.py

```diff
@@ -685,7 +720,8 @@ def write_to_hdf(self, fp, template_hash, root=None):
         else:
             root = '%s/'%(root)
         group = '%scompressed_waveforms/%s' %(root, str(template_hash))
-        for param in ['amplitude', 'phase', 'sample_points']:
+        for param in ['amplitude', 'phase', 
+                      'amplitude_freq_points', 'phase_freq_points']:
```

Comment:
I haven't run compress bank on the full bank yet, so I'm not 100% certain, but based on runs on sub-banks, I expect the full O1 bank to be ~2GB. 25% of that isn't huge, but I wanted to try to keep it as small as possible.